README.md

```diff
@@ -1 +1,17 @@
 # gollum-pow
+
+## Installation
+
+```bash
+export repo_name="my_website"            # set host name
+export repo_path="~/repos/my_website/"   # set content path (ensure '/' at end)
+cd ~/.pow
+mkdir "$repo_name"
+cd "$repo_name"
+git clone https://github.com/chulkilee/gollum-pow.git .
```

Comment:
It may work to place gollum-pow itself under `~/.pow`, but [pow documentation](http://pow.cx/manual.html) only covers making a symlink instead.